src/index.jsx

```diff
@@ -0,0 +1,56 @@
+/* @jsx React.createElement */
+
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import TodoPage from './TodoPage';
+
+const isEmpty = (text) => text.length === 0;
+
+function App() {
+  const [state, setState] = useState({
+    todoTitle: '',
+    todoList: [],
+  });
+
+  const { todoTitle, todoList } = state;
+
+  function handleChangeTitle(text) {
+    setState({
+      todoTitle: text,
+      todoList,
+    });
+  }
+
+  function handleClickAdd() {
+    if (isEmpty(todoTitle)) return;
+
+    setState({
+      todoTitle: '',
+      todoList: [...todoList, { id: Date.now(), title: todoTitle }],
+    });
+  }
+
+  function handleClickDelete(todoItem) {
+    setState({
+      todoTitle,
+      todoList: todoList.filter((todo) => todo.id !== todoItem.id),
+    });
+  }
+
+  return (
+    <div>
+      <TodoPage
+        todoTitle={todoTitle}
+        todoList={todoList}
+        handleChangeTitle={handleChangeTitle}
+        handleClickAdd={handleClickAdd}
+        handleClickDelete={handleClickDelete}
```

Comment:
```suggestion
        onChangeTitle={handleChangeTitle}
        onClickAdd={handleClickAdd}
        onClickDelete={handleClickDelete}
```

핸들러를 부착시키는 곳은 onXXX 이 올바른 표기입니다:)

Reply:
넵, 표기법 확실히 지키겠습니다 : )